### PR TITLE
Advanced transaction modal design fixes

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [x] Tested on small Android

photo_2022-02-23 12 13 29

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201611015895049

### CODE REVIEW

File: src/components/modals/TransactionAdvanceDetails.js [hunk 1]
```diff
@@ -131,7 +131,14 @@ class TransactionAdvanceDetailsComponent extends PureComponent<Props> {
...
             )}
             {signedTx != null && signedTx !== '' ? (
-              <Tile type="copy" title={s.strings.transaction_details_advance_details_raw_txbytes} body={signedTx} />
+              <Tile
+                type="copy"
+                title={s.strings.transaction_details_advance_details_raw_txbytes}
+                body={signedTx}
+                numberOfLines={2}
+                ellipsizeMode="tail"
+                disableFontScaling
```
Comment (amanalili) [inline src/components/modals/TransactionAdvanceDetails.js:140]: remove disableFontScaling

File: src/components/themed/Tile.js [hunk 2]
```diff
@@ -18,7 +18,12 @@ type OwnProps = {
   onPress?: () => void,
   title: string,
   type: 'copy' | 'editable' | 'questionable' | 'loading' | 'static' | 'touchable',
-  contentPadding?: boolean
+  contentPadding?: boolean,
+
+  // Body/Text related props
```
Comment (amanalili) [inline src/components/themed/Tile.js:23]: maximumHeight: 'small' | 'medium' | 'large'

small: 2, ellipsis
medium: 3, ellipsis
large: unli

**Comment by amanalili** [comment]
@thehobbit85 Check the pics when disableFontScaling is enabled/disabled

Font Scaling Enabled
photo_2022-03-07 21 53 42

Font Scaling Disabled
photo_2022-03-07 21 53 54

**Review by thehobbit85 — APPROVED** [review]

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**